apkid/rules/dex/obfuscators.yara

```diff
@@ -169,4 +169,17 @@ rule arxan_multidex : obfuscator
     not arxan
 }
 
+rule allatori_demo : obfuscator
+{
+  meta:
+    description = "Allatori demo"
+    url         = "http://www.allatori.com/features.html"
+    author      = "Eduardo Novella"
+
+  strings:
+    $s = "ALLATORIxDEMO"
```

Comment:
Is there any way to make this more specific? In the code you posted, it looked like it was used for method names, so maybe the string could be `;->ALLATORIxDEMO(`

Are there any other things this obfuscator does that we can capture with a rule here? Maybe some static decryption method or hard coded values? Any added files to the APK?

Reply:
OK, fair enough. Although I have not tested, it is already pushed. 

I do not know anything else now. No assets files added to the APK that I am aware of.

Reply:
The string ` ;->ALLATORIxDEMO(` does not match the rule @CalebFenton. Perhaps we can add the null terminator to increase the length.